cli/src/commands/council/fundBudget.ts

```diff
@@ -0,0 +1,48 @@
+import { flags } from '@oclif/command'
+import chalk from 'chalk'
+import MembershipsCommandBase from '../../base/MembershipsCommandBase'
+
+export default class FundBudget extends MembershipsCommandBase {
+  static description = 'Fund council budget by some member.'
+  static flags = {
+    amount: flags.integer({
+      required: true,
+      description: 'Funding amount',
+    }),
+    rationale: flags.string({
+      required: true,
+      description: 'Reason of funding the budget',
+    }),
+    ...MembershipsCommandBase.flags,
+  }
+
+  async run(): Promise<void> {
+    const { amount, rationale } = this.parse(FundBudget).flags
+
+    // Context
+    const member = await this.getRequiredMemberContext()
+    const keypair = await this.getDecodedPair(member.membership.controllerAccount)
+
+    this.jsonPrettyPrint(JSON.stringify({ amount, rationale }))
+
+    await this.requireConfirmation('Do you confirm the provided input?', true)
+
+    const result = await this.sendAndFollowNamedTx(keypair, 'council', 'fundCouncilBudget', [
+      member.id,
+      amount,
```

Comment:
This is the amount in base units (HAPIs), to make it more convenient it might be better to have input to be a decimal input in JOY and convert it to a BN representing.

If this is the convention for other commands that take a balance amount argument, then better keep it consistent.